nixos/modules/system/boot/plymouth.nix

```diff
@@ -9,7 +9,7 @@ let
 
   cfg = config.boot.plymouth;
 
-  nixosBreezePlymouth = pkgs.breeze-plymouth.override {
+  nixosBreezePlymouth = pkgs.plasma5.breeze-plymouth.override {
```

Comment:
Didn't test this on 20.09 but on unstable this needs to be plasma5Packages.breeze-plymouth.

Reply:
The packages from `plasma5` were moved to `plasma5Packages` in this commit: https://github.com/NixOS/nixpkgs/commit/b209617ff0a8f26ea1d1fe3716a7fa717a50dd95.

I've built NixOS 20.09 with this PR and plasma enabled and can confirm it works.